kf_lib_data_ingest/config.py

```diff
@@ -19,3 +19,5 @@
 DEFAULT_LOG_OVERWRITE_OPT = True
 
 INGEST_PKG_TEMPLATE_NAME = 'my_study'
+
+DEFAULT_ID_CACHE_FILENAME = 'id_cache.json'
```

Comment:
Let's call all of these IDs UIDs or Service UIDs instead. We have a proliferation of ambiguously named ID fields

Reply:
I know this is a long name, but its self-explanatory. What if I call this file `source_uid_to_target_uid_cache.json`?

Also, I made a separate issue to address the naming of the ID fields: https://github.com/kids-first/kf-lib-data-ingest/issues/178

Reply:
uid_cache sounds good 👍

Reply:
Can we do `uid_cache.json` bc its a cache of target uids and also synthetic uids (currently known as unique keys)

Reply:
go for it